### PR TITLE
Feature/pov 58 create an endpoint for team creation

### PR DESCRIPTION
**Pull request by njokc10**

### CODE REVIEW

**Review by AljazKnific — APPROVED** [review]

File: src/data-api/teams/admin/dto/create-team.admin.body.dto.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { IsString, IsOptional } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateTeamAdminBodyDto {
+  @ApiProperty({ description: 'Team name' })
```
Comment (evapincin) [inline src/data-api/teams/admin/dto/create-team.admin.body.dto.ts:5]: do we intend to write examplesfor api properties?

**Review by evapincin — APPROVED** [review]

File: src/data-api/teams/user/dto/create-team.user.res.dto.ts
[the remark this answers is not in the document]
Comment (VitoDrofenik) [reply src/data-api/teams/user/dto/create-team.user.res.dto.ts:1]: What do you mean by that? They are exposed

File: src/data-api/teams/admin/dto/create-team.admin.body.dto.ts (hunk 1 above)
Comment (VitoDrofenik) [reply src/data-api/teams/admin/dto/create-team.admin.body.dto.ts:5]: I am unsure. I remember that we talked about it but I can't remember coming to a conclusion

File: src/data-api/teams/user/dto/create-team.user.res.dto.ts
[the remark this answers is not in the document]
Comment (AljazKnific) [reply src/data-api/teams/user/dto/create-team.user.res.dto.ts:1]: True, my bad

**Pull request merged**